### DISCUSSION TITLE
Retried jobs missing in list of pipeline jobs

### DISCUSSION THREAD

**Discussion by FlexOr2** [python-gitlab/python-gitlab#1898] [Q&A] [answered]
Hi,

if i use the command `pipeline.jobs.list(all=True)` it seems to me that not all jobs of the pipeline are retrieved. More precisely the retried jobs of the pipeline are missing.

To get the missing jobs i currently use `project.jobs.list(all=True,as_list=False)` as an alternative.
Unfortunately this takes a long time, so i would like to filter the request to only get the jobs of a specific pipeline.

Is that possible or am i doing something wrong in general?

**Comment by JohnVillalovos** [comment, marked as answer]
See https://docs.gitlab.com/ee/api/jobs.html#list-pipeline-jobs

Use `include_retried=True`

**Reply by FlexOr2** [reply]
Thank you very much !

`pipeline.jobs.list(all=True, include_retried=True)  ` 

works !
